### PR TITLE
add support for model parameters in prompt templates

### PR DESCRIPTION
**Pull request by abubakarsohail**

### CODE REVIEW

File: promptlayer/prompts/prompts.py [hunk 1]
```diff
@@ -10,24 +10,39 @@
 )
 
 
-def get_prompt(prompt_name, langchain=False, version: int = None, label: str = None):
+def get_prompt(
+    prompt_name,
+    langchain=False,
+    version: int = None,
+    label: str = None,
+    include_parameters: bool = False,
```
[the remark this answers is not in the document]
Comment (Jped) [reply promptlayer/prompts/prompts.py:18]: @abubakarsohail is this comment by droid true? Or are you providing a default value False?

File: promptlayer/prompts/prompts.py [hunk 2]
```diff
@@ -10,47 +10,58 @@
...
+            "model_provider": prompt["model_provider"],
+            "model": prompt["model"],
+            "parameters": prompt["parameters"],
+        }
+    return prompt_template
 
 
-def publish_prompt(prompt_name, tags=[], commit_message=None, prompt_template=None):
+def publish_prompt(
+    prompt_name, tags=[], commit_message=None, prompt_template=None, model=None
```
Comment (Jped) [inline promptlayer/prompts/prompts.py:49]: @droid what should we name the model argument?

it is a dictionary that takes `model`, `parameters` and `model_provider`

File: promptlayer/prompts/prompts.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (abubakarsohail) [reply promptlayer/prompts/prompts.py:18]: @Jped already providing a default value for `include_parameters`. If you notice the suggestion it made, it's just rewriting the same line again. Don't know why it still added this comment

File: promptlayer/prompts/prompts.py (hunk 2 above)
Comment (abubakarsohail) [reply promptlayer/prompts/prompts.py:49]: @factory-droid what should we name the model argument?

it is a dictionary that takes model, parameters and model_provider

Comment (abubakarsohail) [reply promptlayer/prompts/prompts.py:49]: @Jped should we rename it to `model_parameters`?

Comment (Jped) [reply promptlayer/prompts/prompts.py:49]: yes, that is a good idea.

**Review by Jped — APPROVED** [review]
There are a few places where we need to change the name from `model` to `model_ parameters` once that is changed we should be done

**Pull request merged**